bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteStringUtil.java

```diff
@@ -12,6 +12,29 @@
  * @version $Id: $Id
  */
 public final class BigtableZeroCopyByteStringUtil {
+
```

Comment:
So, AFAICT this code is doing the same thing:

https://github.com/tsuna/asynchbase/commit/b55d9092efa7b13de7022cd02aa2a465971056ca

In fact, this is sort of how HBaseBigtableZeroCopyUtil used to work. Can you explain why that approach doesn't work here? Presumably it breaks in protobuf 3.0 but it wasn't immediately obvious to me how/why?